test/function_test.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+"""
+>>> import os
+>>> from nava import play
+>>> play(os.path.join("others", "test.wav"))
```

Comment:
I couldn't find the exact place in which we test the correct performance of the `play` function. (As you said, comparing waveforms and etc.)

Reply:
Github Action does not install any drivers by default. After installing the driver, if the system succeeds in playing the file, no error will occur. That's enough at the moment.